### PR TITLE
Updated SEOSpider to version 16.0.

### LINKED ISSUE #28684: [ScreamingFrog.SEOSpider version 15.2] Installer hash does not match

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

Resolves #28684 (probably)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28712)

### CODE REVIEW

**Review by OfficialEsco — APPROVED** [review]

File: manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.installer.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Created using wingetcreate
-
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 PackageIdentifier: ScreamingFrog.SEOSpider
-PackageVersion: 15.2
+PackageVersion: "15.2"
```
Comment (OfficialEsco) [inline manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.installer.yaml:3]:
```suggestion
PackageVersion: "16.0"
```

File: manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.locale.en-US.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
-# Created using wingetcreate
-
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
 PackageIdentifier: ScreamingFrog.SEOSpider
-PackageVersion: 15.2
+PackageVersion: "15.2"
```
Comment (OfficialEsco) [inline manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.locale.en-US.yaml:3]:
```suggestion
PackageVersion: "16.0"
```

File: manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-# Created using wingetcreate
-
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
 PackageIdentifier: ScreamingFrog.SEOSpider
-PackageVersion: 15.2
+PackageVersion: "15.2"
```
Comment (OfficialEsco) [inline manifests/s/ScreamingFrog/SEOSpider/15.2/ScreamingFrog.SEOSpider.yaml:3]:
```suggestion
PackageVersion: "16.0"
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Uhoh i went too fast

**Review by OfficialEsco — DISMISSED** [review]
Thaaanks

File: manifests/s/ScreamingFrog/SEOSpider/16.0/ScreamingFrog.SEOSpider.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: ScreamingFrog.SEOSpider
+PackageVersion: "16.0"
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/s/ScreamingFrog/SEOSpider/16.0/ScreamingFrog.SEOSpider.installer.yaml:5]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**